text/0061-error-format-and-definitions.md

```diff
@@ -1474,6 +1474,31 @@ HTTP Code: `400 Bad Request`
 
 ---
 
+## invalid_search_vector
+
+`Synchronous`
+
+### Context
+
+This error occurs for the listed reasons:
+- if a value with a different type than `Array of Float` or `null` for `vector` is specified.
```

Comment:
Is `Array of Float` an existing type?

Reply:
Is there a point in setting the value of the `vector` field to null?

Reply:
@alallema 

> An array of arrays and an empty array aren't supposed to work too?

Not at search time; it only accepts an array made of float (no sub-array dimensions) or `null` the default one (equivalent to not sending a vector query)

Reply:
cc @kero

What is the default value of the `vector` field at search-time? `null` or `[]`.

If we don't accept `[]`, the default value specified should be `null` or the field right?

Reply:
> Is there a point in setting the value of the vector field to null?
Depending on your code if you set a query and want to be sure not to use the vector.
```[2023-07-13T16:05:55Z WARN  meilisearch::search] Ignoring the query string `q` when used with the `vector` parameter.```